sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -191,50 +190,544 @@ def get_next(next_link=None):
         return ItemPaged(get_next, extract_data)
 
     @distributed_trace
-    def get_repository(self, repository_name, **kwargs):
-        # type: (str, Any) -> ContainerRepository
-        """Get a Container Repository object
+    def get_repository_properties(self, repository, **kwargs):
+        # type: (str, **Any) -> RepositoryProperties
+        """Get the properties of a repository
 
-        :param str repository_name: The repository to create a client for
-        :returns: :class:`~azure.containerregistry.ContainerRepository`
-        :raises: None
+        :param str repository:
+        :returns: :class:`~azure.containerregistry.RepositoryProperties`
+        :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+        """
+        return RepositoryProperties._from_generated(  # pylint: disable=protected-access
+            self._client.container_registry.get_properties(repository, **kwargs)
+        )
+
+    @distributed_trace
+    def list_manifests(self, repository, **kwargs):
```

Comment:
I called out that all methods have been moved to `ContainerRegistryClient` and `ContainerRepository` and `RegistryArtifact` have been removed. Otherwise it might be too verbose, I'll talk with the v-team and Johan/anna